ujt/constants.py

```diff
@@ -116,6 +135,44 @@
     },
 ]
 
+# Generate the necessary gradient styles for change over time feature
+subclasses = [
+    Status.Name(Status.STATUS_HEALTHY),
+    f"{Status.Name(Status.STATUS_HEALTHY)}_{IMPROVED_SUBCLASS}",
+    f"{Status.Name(Status.STATUS_HEALTHY)}_{WORSENED_SUBCLASS}",
+    Status.Name(Status.STATUS_WARN),
+    f"{Status.Name(Status.STATUS_WARN)}_{IMPROVED_SUBCLASS}",
+    f"{Status.Name(Status.STATUS_WARN)}_{WORSENED_SUBCLASS}",
+    Status.Name(Status.STATUS_ERROR),
+    f"{Status.Name(Status.STATUS_ERROR)}_{IMPROVED_SUBCLASS}",
+    f"{Status.Name(Status.STATUS_ERROR)}_{WORSENED_SUBCLASS}",
+    NO_DATA_SUBCLASS,
+]
+subclass_colors = [
+    HEALTHY_COLOR,
+    HEALTHY_IMPROVED_COLOR,
+    HEALTHY_WORSENED_COLOR,
+    WARN_COLOR,
+    WARN_IMPROVED_COLOR,
+    WARN_WORSENED_COLOR,
+    ERROR_COLOR,
+    ERROR_IMPROVED_COLOR,
+    ERROR_WORSENED_COLOR,
+    NO_DATA_COLOR,
+]
+for before_subclass, before_color in zip(subclasses, subclass_colors):
+    for after_subclass, after_color in zip(subclasses, subclass_colors):
+        BASE_CYTO_STYLESHEET.append(
+            {
+                "selector": f".{before_subclass}_{after_subclass}",
+                "style": {
+                    "background-fill": "linear-gradient",
+                    "background-gradient-stop-colors": f"{before_color} {after_color}",
+                    "background-gradient-direction": "to-right",
+                },
+            }
+        )
```

Comment:
Should this be in a function? We dynamically generate these combinations (e.g. STATUS_HEALTHY_STATUS_ERROR -> gradient with HEALTHY_COLOR and ERROR_COLOR), but they are constant. I feel like even though it's dynamically generated we shouldn't have to call a function because there's no guarantee that caller would call the setup function.

Reply:
leaving as is is probably fine.
you can always spit out the actual hex values and then hardcode them into the constants too.